docs/20_tutorial.md

```diff
@@ -1,64 +1,245 @@
 \pagebreak
 
-Run Book outline
-========
+# Run Book 
 
-***should the run book deal with the SOS establishing the risk
-limit, or any of their pre-audit responsibilities like the selection
-of which races to audit?
+## Introduction
 
-Brief introduction to RLAs in non-technical, human friendly language.
-At the county level, something along the lines of "This tool is going
-to help you cross reference votes recorded on paper ballots with how 
-the computers read those ballots, to determine whether the computers
-did their job accurately…” Try to put their minds at ease about the
-process. :)
+### What the RLA Tool Does
+
+The Risk-Limiting Audit (RLA) tool, developed by Free & Fair 
+for the Colorado Department of
+State for use in elections from November 2017 forward, supports a 
+Risk-Limiting Audit as required by Colorado statute and 
+as described in the Colorado Secretary of State's Rule 25 for
+elections. RLA-related terms and processes are defined and prescribed in Rule 25.
+*** citation for Rule 25.
+
+The RLA Tool enables the Secretary of State and each individual 
+County to carry out a Risk-Limiting Audit. The Secretary of State can
+enter the risk limit, the random seed and the contests that drive
+the audit process. The Secretary of State can also identify 
+contests that should go to a full hand count. Counties can upload
```

Comment:
For the User Manual I agree. OK with you if the Run Book is more brass-tacks, aimed not to teach people about RLAs but to enable people involved in RLAs (and so presumably at least a bit knowledgeable) to use the computer tool to do what they need to do?